omod/src/main/java/org/openmrs/module/util/SingleSpaConstants.java

```diff
@@ -20,6 +20,10 @@
 	
 	public static final String GP_KEY_SPA_BASE_URL = "spa.baseUrl";
 
+    public static final String GP_BODY_CONTENT_URL = "spa.bodyContentUrl";
```

Comment:
Please use tabs instead of spaces in this file to match the other code here.

Reply:
This should be okay now.